src/search/search.module.ts

```diff
@@ -0,0 +1,23 @@
+// modules
+import { NgModule } from "@angular/core";
+import { FormsModule } from "@angular/forms";
+import { CommonModule } from "@angular/common";
+
+// imports
+import { Search } from "./search.component";
+
+@NgModule({
+	declarations: [
+		Search
+	],
+	exports: [
+		Search
+	],
+	imports: [
+		FormsModule,
+		CommonModule
```

Comment:
You need `I18nModule` here for build to pass.
```suggestion
		CommonModule,
		I18nModule
```